src/main/java/lotto/view/LottoOutputView.java

```diff
@@ -0,0 +1,75 @@
+package lotto.view;
+
+import lotto.model.domain.*;
+
+import java.util.stream.Collectors;
+
+public final class LottoOutputView {
+
+    private static final LottoOutputView INSTANCE = new LottoOutputView();
+
+    private LottoOutputView() {
+    }
+
+    public static LottoOutputView getInstance() {
+        return INSTANCE;
+    }
+
+    public void printBuyingLotto(final Lottos lottos) {
+        lottos.getLottos().forEach(
+                lotto -> System.out.println("[" + toLottoOutput(lotto) + "]")
+        );
+        System.out.println();
+    }
+
+    public void printLottoResult(final LottoResult lottoResult) {
+        System.out.println();
+        System.out.println("당첨 통계");
+        System.out.println("----------------");
+
+        for (final Rank rank : Rank.values()) {
+            printRankResult(lottoResult, rank);
+        }
+
+        final double profitRate = lottoResult.getProfitRate();
+        System.out.printf("총 수익률은 %.2f 입니다. ", profitRate);
+        System.out.println(profitRateResult(profitRate));
+    }
+
+    public void printSizeOfLottos(final Lottos lottos) {
+        final int lottoSize = lottos.getLottos().size();
+        System.out.println(lottoSize + "개를 구매했습니다.");
+    }
+
+    public void printChangeOfPurchase(final LottoMoney change) {
+        System.out.printf("거스름돈은 %d원 입니다.", change.getValue()).println();
+    }
+
+    private void printRankResult(final LottoResult lottoResult, final Rank rank) {
+        System.out.printf("%d개 일치", rank.getMatchCount());
+        System.out.printf("%s ", printBonusBall(rank));
+        System.out.printf("(%d원)", rank.getPrize());
+        System.out.printf(" - %d", lottoResult.getCount(rank)).println();
```

Comment:
메시지 포맷이 한 눈에 보이는 코드도 좋겠어요. 실제 코드 작업은 읽는데 80%, 코드 작성엔 20% 정도이므로 남이 이 코드를 읽을 때 내용을 쉽게 파악할 수 있을까? 에 중점을 두고 코딩을 하는게 좋은 코드를 만드는 습관이라 생각합니다.

```suggestion
// ex. 5개 일치, (보너스 볼 일치) (20,000,000원) - 1
private static final String LOTTO_RESULT_MESSAGE_FORMAT = "%d개 일치%s(%d원) - %d";

private void printRankResult(final LottoResult lottoResult, final Rank rank) {
    int matchCount = rank.getMatchCount();
    String bonusBallMessage = printBonusBall(rank);
    long prize = rank.getPrize();
    int count = lottoResult.getCount(rank);

    System.out.printf(LOTTO_RESULT_MESSAGE_FORMAT, matchCount, bonusBallMessage, prize, count);
    System.out.println();
}

```